tests/platform/windows/windows-firefox-testSpec.js

```diff
@@ -0,0 +1,105 @@
+/*
+GPII Integration and Acceptance Testing
+
+Copyright 2014 Emergya
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+"use strict";
+var fluid = require("universal"),
+    gpii = fluid.registerNamespace("gpii");
+
+gpii.loadTestingSupport();
+
+fluid.registerNamespace("gpii.tests.windows.firefox");
+
+
+
+gpii.tests.windows.firefox = [
+    {
+        name: "Acceptance test for background color change in firefox",
+        userToken: "chrome_high_contrast",
```

Comment:
It would be better if we add a few firefox-specific NP sets rather than re-using the NP sets for chrome.